src/cbmc/cbmc_parse_options.cpp

```diff
@@ -1137,6 +1160,10 @@ void cbmc_parse_optionst::help()
     HELP_TIMESTAMP
     " --write-solver-stats-to json-file\n"
     "                              collect the solver query complexity\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    " --use-rra abstraction-file\n"
+    // NOLINTNEXTLINE(whitespace/line_length)
+    "                              abstract the program as specified in the abstraction-file"
```

Comment:
Too generic, how about `apply the replication reducing abstraction configured by the abstraction-file`?

Reply:
This option is removed.